llvm/lib/Support/CommandLine.cpp

```diff
@@ -400,6 +400,22 @@ class CommandLineParser {
   SubCommand *LookupSubCommand(StringRef Name, std::string &NearestString);
 };
 
+template <typename T, T TrueVal, T FalseVal>
+bool parseBool(Option &O, StringRef ArgName, StringRef Arg, T &Value) {
```

Comment:
move the function outside of the unnamed namespace and add `static` per https://llvm.org/docs/CodingStandards.html#anonymous-namespaces

The majority of commits omit the trailing full stop in the subject line.

Reply:
Done.